specs/proposals.md

```diff
@@ -24,13 +27,23 @@ are part of later specification milestones.
 
 [cannon]: https://github.com/ethereum-optimism/cannon
 
-## Proposing L2 output commitments
+## Constants
+
+| Name                   | Value  |
+| ---------------------- | ------ |
+| `SUBMISSION_FREQUENCY` | `100?` |
 
-The proposer's role is to construct and submit output commitments on a configurable interval to a contract on L1.
+## Proposing L2 output commitments
 
-TODO: describe integration with rollup node and L2 execution engine (see PR #179).
+The proposer's role is to construct and submit output commitments on a configurable interval to a contract on L1, which
+it does by running the [L2 output submitter](../l2os/) service (AKA L2OSS). This service periodically queries the rollup
+ node's [`optimism_outputAtBlock` rpc method](./rollup-node.md#l2-output-rpc-method) for the latest output root derived
+ from the latest [finalized](rollup-node.md#finalization-guarantees) L1 block. The construction of this output root is
+ described [below](#l2-output-commitment-construction).
```

Comment:
Pedantic: is it really necessary to call it a "service"? "L2 output submitted" seems quite descriptive enough to me, not sure what "service" adds.

Reply:
Mostly I don't like the term 'OS' it confuses me.

Reply:
Fair point but maybe it's better to rename `l2os` to `l2output`?

Reply:
just going with l2os so we can move forward